src/script/index.js

```
@@ -1 +1,79 @@
+const hamburgerIconClass = 'icon-hamburger';
+const menuIconClass = 'menu-icon';
+const toggleMenuClass = 'navbar__menu-toggle';
+
+const navlinksContainerClass = 'navlinks__container';
+const buttonContainerClass = 'navbar__button-container';
+const navbarMenuActiveClass = 'navbar__navigation-drawer-navlinks--active';
+const navbarButtonActiveClass =
+    'navbar__navigation-drawer-button-container--active';
+const bodyScrollStop = 'body--stop-scrollY';
+
+const menuIcon = document.querySelector('.' + menuIconClass);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)